public/data_load/enhanced-table-request-handler.js

```diff
@@ -85,8 +84,10 @@ export async function enhancedTableRequestHandler ({
     });
   }
 
-  inspectorAdapters.requests = new RequestAdapter();
-  inspectorAdapters.data = new DataAdapter();
+  //inspectorAdapters.requests is already initialized
+  //inspectorAdapters.requests = new RequestAdapter();
+  // DataAdapter is not present in Kibana 7.12. Is this necessary?
+  //inspectorAdapters.data = new DataAdapter();
```

Comment:
it's maybe somewhat necessary, because "Inspect" view don't show anymore data into "View: data" :(
You should take inspiration from Kibana 7.12 "esaggs" component.

Reply:
After updating `courier.ts` and `enhanced-table-request-handler.js` to be as similar as possible to `esaggs`, Data tab of the inspect window was still empty.

Looking at other visualizations in Kibana I found some lines inside the expression functions:
```js
    if (inspectorAdapters?.tables) {
      inspectorAdapters.tables.logDatatable('default', input.tables[0]);
    }
```
It seems that `logDataTable`is the responsible to pass that data to the Inspect window. I'm using `input.tables[0]`. Can that `tables` array have more than one entry? If that is the case maybe `logDataTable`should be called for every table of the array. What do you think?

Reply:
I made some more text and I have fixed it passing to `logDataTable` the result of the requestHandler instead of the result of the responseHandler. I think that will work event if there is more than one table after the responseHandler runs. Please see this [commit](https://github.com/cpiment/kibana-enhanced-table/commit/df09017ca51648335abb0b0c29602ef8f213dd13)

Reply:
Nice!
I just tested it, it works well now!